adapters/repos/db/index.go

```diff
@@ -289,7 +289,7 @@ func (i *Index) putObject(ctx context.Context, object *storobj.Object,
 	if i.replicationEnabled() {
 		if replProps == nil {
 			replProps = &additional.ReplicationProperties{
-				ConsistencyLevel: string(replica.All),
+				ConsistencyLevel: string(replica.Quorum),
```

Comment:
Maybe it would make sense to have  function like:
```go
package replica

import "github.com/weaviate/weaviate/entities/additional"

func DefaultProperties() *additional.ReplicationProperties {
    return &additional.ReplicationProperties{
        ConsistencyLevel: string(Quorum),
    }
}
```

so we can simplify this in all these places, and update the defaults in only one place

Reply:
Then throughout `index.go` we can simply have:

```go
if replProps == nil {
    replProps = replica.DefaultProperties()
}
```

Reply:
Makes sense to me
Since only db/index.go makes use of the default consistency level I've created similar function in index.go for this same purpose.

Reply:
Cool, works for me 👍